ImageClassification/ILSVRC2012/Alexnet/alexnet_neon.py

```diff
@@ -105,15 +135,15 @@
 
 # configure callbacks
 valmetric = TopKMisclassification(k=5)
-callbacks = Callbacks(model, eval_set=test, metric=valmetric, **args.callback_args)
+callbacks = Callbacks(model, eval_set=val, metric=valmetric, **args.callback_args)
 
 if args.model_file is not None:
     model.load_params(args.model_file)
```

Comment:
model.load_params(args.model_file) should work. Why does resnet_eval.py in cifar10 does not support this. 
We can see alexnet support this.